packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -939,6 +939,20 @@
                 }
             ]
         },
+        {
+            "name": "amazonq_wsOrphanedDocuments",
```

Comment:
In addition to my other comment, can we move this to the common definition in https://github.com/aws/aws-toolkit-common

Reply:
https://github.com/aws/aws-toolkit-common/pull/856

Reply:
@jpinkney-aws should we merge common first?